gaurun/client.go

```diff
@@ -5,7 +5,7 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/mercari/gcm"
+	"github.com/mercari/gaurun/service/google"
```

Comment:
When you say "should", cloud you give a reason for that since I named it with a reason (it's not only GCM anymore as you said and much more make sense).